src/components/Nav.jsx

```diff
@@ -5,17 +5,71 @@ import brigadeLogo from '../images/cfc_logo_2021.svg'
 import slackLogo from '../images/slack.svg'
 import githubLogo from '../images/github.svg'
 import meetup from '../images/meetup.svg'
+import styled from 'styled-components'
+
+const StyledNav = styled.nav`
+  height: 6rem;
+  padding: 1.5rem 6rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: white;
```

Comment:
Let's use a CSS variable for this color. It is highly unlikely on this project, but it's a good habit to never rely on a project never changing its default colors.